### PR TITLE
Support safety_settings feature for Vertex AI

### PR DESCRIPTION
**Pull request by gaya3-vijayakumar**
## What

Support safety_settings feature for Vertex AI. - This involves:
- updation of llama-index to 0.10.38
- updation of google related libraries to keep google-generativeai code to latest
- - updation of milvus to resolved a mypy error (actually milvus is not supported)

## Why

To allow Gemini users to tune the responses

## How

Through integration of APIs supported by llama-index.

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

- Tested creation and edit of Gemini adapter using Vertex AI with safety settings

## Screenshots

image

image

image

Existing adapters without the settings will respond as

image

Post edit/test connection with the newly added params, the same adapter on Get details will respond as

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: src/unstract/adapters/llm/vertex_ai/src/vertex_ai.py [hunk 1]
```diff
@@ -54,28 +80,102 @@ def get_llm_instance(self) -> LLM:
...
+        safety_settings_user_config: dict[str, str] = self.config.get(
+            SafetySettingsConstants.SAFETY_SETTINGS,
+            safety_settings_default_config,
+        )
+
+        vertex_safety_settings: dict[HarmCategory, HarmBlockThreshold] = (
+            self._get_vertex_safety_settings(safety_settings_user_config)
+        )
+
+        logger.info(f"Safety settings: {vertex_safety_settings}")
```
Comment (hari-kuriakose) [inline src/unstract/adapters/llm/vertex_ai/src/vertex_ai.py]: @gaya3-zipstack Let's remove this info log from here.

**Review by hari-kuriakose — APPROVED** [review]
@gaya3-zipstack LGTM overall.

File: src/unstract/adapters/llm/vertex_ai/src/vertex_ai.py (hunk 1 above)
Comment (gaya3-vijayakumar) [reply src/unstract/adapters/llm/vertex_ai/src/vertex_ai.py]: Done

File: src/unstract/adapters/ocr/register.py [hunk 2]
```diff
@@ -46,4 +44,4 @@ def _build_adapter_list(
                         Common.METADATA: metadata,
                     }
         except ModuleNotFoundError as exception:
-            logger.error(f"Error while importing ocr adapters : {exception}")
+            logger.warning(f"Unable to ocr adapters : {exception}")
```
Comment (tahierhussain) [inline src/unstract/adapters/ocr/register.py]: @gaya3-zipstack NIT: Looks like the message is not clear. Shouldn't it be "Unable to import ocr adapters"?

**Review by tahierhussain — APPROVED** [review]
Overall, LGTM except a minor change related to log message

File: src/unstract/adapters/ocr/register.py (hunk 2 above)
Comment (gaya3-vijayakumar) [reply src/unstract/adapters/ocr/register.py]: Thank you. Yes. Changed now.

**Pull request merged**